app/routes/invitations.rb

```diff
@@ -3,11 +3,13 @@ module Routes
     class Invitations < Core
       post '/teams/:slug/invitations' do |slug|
         please_login
+        team = Team.find_by(slug: params[:slug])
+        valid_invitee = (params[:usernames].split(",") - team.members.map(&:username)).join(",")
```

Comment:
I think this check will be done twice now (now here and [here](https://github.com/exercism/exercism.io/blob/3de39152058110ace5b1ea98f3f9915d13480294/lib/exercism/team.rb#L76)).